vp.json

```diff
@@ -0,0 +1,169 @@
+[
+    {
+      "image": "nginx:latest",
```

Comment:
parser expects data to be in this format only from knoxguard, am i correct?

Reply:
Yes, @rksharma95 we are expecting a same response from the knoxguard.